common/changes/@gooddata/sdk-ui-all/lhq-sd-1179_2020-10-23-10-23.json

```diff
@@ -2,7 +2,7 @@
     "changes": [
         {
             "packageName": "@gooddata/sdk-ui-all",
-            "comment": "Load date format for InsightView component",
+            "comment": "The load date format is added to the InsightView component.",
```

Comment:
This is not correct, the idea here is that the dateFormat that the user/workspace has defined is now loaded in the InsightView and used in the visualizations. So the load part is important there.

Reply:
Ahhhh okay! Rewriting...